cmd/kustomize/README.md

```diff
@@ -4,6 +4,7 @@
 [base]: docs/glossary.md#base
 [declarative configuration]: docs/glossary.md#declarative-application-management
 [demo]: demos/README.md
+[demos]: demos/README.md
```

Comment:
demos and demo point to the same file. Do you want to keep both of them?

Reply:
yes.  only way to do plural

Reply:
it's a markdown thing.  Want to be able to use \[demo\] and \[demos\] in text as links.